docs/reference/remap/to_string.cue

```diff
@@ -1,21 +1,23 @@
 package metadata
 
 remap: functions: to_string: {
```

Comment:
We'll probably (separately from this PR) want to update the function documentation definition to allow us to document if a function can be fallible or not.

Reply:
For sure!